src/panel/poi/blocks/Stars.jsx

```diff
@@ -0,0 +1,35 @@
+/* global _n */
```

Comment:
We should use the hook for this function too, it's easy to export it from `useI18n` like we did for `_` (should have done it when creatinf the hook, sorry)

Reply:
Sure, what hook do you mean, usei18n()._n ?
and if we add it, does it mean that this comment /* global _n */ will no longer be necessary?
Thanks

Reply:
You can export `_n` the same way it's already done with `_` in the hook here https://github.com/Qwant/erdapfel/blob/master/src/hooks/useI18n.js#L10-L15.
That way you can import both in one round:
```js
const { _, _n } = useI18n();
``` 
And yes, as a result, the global component will have to be removed, as the linter won't complain anymore.
That's the point of it, reducing the use of globals in the app by scoping them. For now the hook acts just as a relay, as you see it's re-exporting `window` globals. But each time we use it it's one less global, so when there are no more it will be a single point to migrate to a better system.